source/SymWorld.h

```diff
@@ -205,6 +206,45 @@ class SymWorld : public emp::World<Organism>{
 
     return file;
   }
+  emp::DataFile & SetupSggSymIntValFile(const std::string & filename) {
```

Comment:
For this and the host equivalent, currently you've got it set up so that you're printing out a lot of duplicate information between files (since you're also setting up the standard data files). If you take my reccomendation above to track the full histogram for the donation value, I'd reccomend printing the histbins from that DataNode here, instead of from the SymInt DataNode (since you'll already have those in the sym file).